CMakeLists.txt

```diff
@@ -23,6 +23,25 @@ if(NOT OPENCL_ROOT)
     message(FATAL_ERROR "Can't find opencl")
 endif()
 
+find_path(HCC_ROOT bin/hcc
+    PATH_SUFFIXES
+        hcc
```

Comment:
This should be `bin`.

Reply:
oh, but hcc is located in both /opt/rocm/bin and /opt/rocm/hcc/bin. I guess we could add both path suffixes

Reply:
actually clang is not installed to /opt/rocm/bin, so we cannot use suffix bin, since /opt/rocm/bin won't have clang